include/dice/template-library/defer.hpp

```diff
@@ -95,6 +101,12 @@ namespace dice::template_library {
 	}
 
 	namespace detail {
+		/**
+		 * Helper types and functions whose only purpose it to improve
+		 * the syntax of the provided `DEFER*` macros to make them seem more like actual language
+		 * constructs rather than macros (see macro definitions below).
+		 */
```

Comment:
```suggestion
	/**
	 * Helper types and functions whose only purpose it to improve
	 * the syntax of the provided `DEFER*` macros to make them seem more like actual language
	 * constructs rather than macros (see macro definitions below).
	 */
	namespace defer_detail {
```